test/box-luatest/gh_8036_create_space_with_explicit_and_implicit_id_test.lua

```diff
@@ -0,0 +1,63 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group('Create space with explicit and implicit id', {
+    {sql = false},
+    {sql = true}
```

Comment:
Parameterization by `sql = true/false` looks weird. Better duplicate the code - the test isn't that big.

Reply:
Done